ebsco/edsapi-koha-plugin
# Cart message fix

Moves logic for adding/removing catalog prefix in bibnumber when adding to cart from JS to native perl.  This resolves the double call of addRecord happening from the event bound by both Perl and EDSScript by only letting the native script bind the event.

To test:
http://mike-koha.ebsco-gss.net/
Search for any item.
Limit to Academic Journals (catalog does not work on test site).
Click "Add to cart".
Note that the message that pops up says "Item added to cart" and does NOT say "Item already in cart".
